packages/features/tasker/tasks/triggerNoShow/common.ts

```diff
@@ -215,7 +215,7 @@ export const prepareNoShowTrigger = async (
   const numberOfHostsThatJoined = hosts.length - hostsThatDidntJoinTheCall.length;
 
   const didGuestJoinTheCall = meetingDetails.data.some(
-    (meeting) => meeting.max_participants < numberOfHostsThatJoined
+    (meeting) => meeting.max_participants > numberOfHostsThatJoined
```

Comment:
This was the logic that caused the issue.

`max_participants` represents the peak number of concurrent participants in a meeting session and If `max_participants` is greater than numberOfHostsThatJoined, it means there must have been at least one guest present